src/heads-up/views/slack.vue

```diff
@@ -25,13 +26,62 @@
 
     <panel-section title="Properties">
       <properties-list>
-        <template v-if="og.title">
-          <dt>og:title</dt>
+        <template>
+          <dt>
+            <p v-if="!og.title">
+              og:title
+            </p>
+            <app-tooltip
```

Comment:
So if I understand this correctly, the tooltip should always render? And then it will either render og:title or an og:title with an icon?

Reply:
Thats rights! Only if something is missing, the icon will show up and is able to be hovered over. If everything works correctly, you will be able to hover over the text og:title